screenly-cast/theme/screenly-cast/assets/js/category_scripts.js

```diff
@@ -0,0 +1,83 @@
+(function()
+{
+    "use strict";
+
+    var article = document.querySelector('article');
+    if (article) {
+        var content = article.querySelector('.content');
+        if (content) {
+            var childs = content.children;
+
+            /*
+             * LET'S START REMOVING CHILD BLOCKS THAT ARE OUT OF VIEW
+             */
+            var i = childs.length;
+            for (i; i>0; i--) {
+                var child = childs[i];
+                if (typeof child === 'object') {
+                    var childRect = child.getClientRects()[0];
+
+                    if (childRect.top > article.clientHeight) {
```

Comment:
The error "Cannot read property 'top' of undefine" as the parent (<section>) has a property display:none. That's why slider doesn't work and revert blank screen.